RecoLocalTracker/SiStripClusterizer/plugins/clusterGPU.cu

```diff
@@ -13,13 +13,409 @@
 #include "ChanLocsGPU.h"
 #include "SiStripRawToClusterGPUKernel.h"
 #include "clusterGPU.cuh"
+#include "localToGlobal.cuh"
```

Comment:
All the kernels etc. for your producer should be in a separate file, we shouldn't mix code specific to different producers in the same file.